### PR TITLE
test/utils/image: remove listx from OWNERS

### PR DESCRIPTION
**Pull request by listx**
Removing myself from OWNERS as I haven't had the bandwidth to go over
these reviews for a long time.

This should have been part of #99718, which has already been merged.

#### What type of PR is this?

/kind cleanup

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by aojea** [comment]
/retest
flakes on 1 sig-network and 3 storage
sad to see you go :wave: 
/lgtm

File: test/utils/image/OWNERS [hunk 1]
```diff
@@ -4,12 +4,10 @@ reviewers:
   - justaugustus
   - luxas
   - mkumatag
-  - listx
   - dims
 approvers:
   - luxas
   - mkumatag
-  - listx
```
Comment (spiffxp) [inline test/utils/image/OWNERS:12]: FWIW we typically move folks to `emeritus_approvers`, it's not consumed by automation.  if you don't care or would rather not be there, feel free to `/hold cancel`

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm
/milestone v1.21
owners-only change
/hold
but I'm adding a hold for @listx to decide whether to update this PR or merge it as-is

**Pull request merged**